src/main.py

```diff
@@ -237,6 +241,56 @@ async def info(interaction: Interaction) -> None:
     await bc.send_stats(interaction)
 
 
+# Preview command
+@client.slash_command(dm_permission=False)
+async def preview(
+    interaction: Interaction,
+    uploaded_file: Attachment = SlashOption(description="The file to submit."),
```

Comment:
```suggestion
    uploaded_file: Attachment = SlashOption(description="The song to submit."),
```

Just so folks don't think cover art needs to be put here, or something.